### DISCUSSION TITLE
Is there a Visual Analyzer available for UnoCSS?

### DISCUSSION THREAD

**Discussion by PaquitoSoft** [unocss/unocss#2901] [Q&A] [answered]
Today I discovered [this two years old article ](https://antfu.me/posts/reimagine-atomic-css) by Anthony Fu where he explains about the background behind building UnoCSS.

One of the things that caught my attention was this [Visual Analyzer feature](https://windicss.org/features/analyzer.html) WindiCSS has and I was wondering if UndoCSS has such a feature.
I don't seem to find it so, if it doesn't, is there any plan to implement it?

I believe this would be a great addition specially for teams using UnoCSS in design system components libraries.

Thanks!

**Comment by zyyv** [comment, marked as answer]
Yes, we had it ([Inspector](https://unocss.dev/tools/inspector#inspector)). And there is a PR(https://github.com/unocss/unocss/pull/2762) will be bring new feature.

**Reply by PaquitoSoft** [reply]
That's awesome!.

Thanks for your answer 👍
